Source/WTF/wtf/ASCIICType.h

```diff
@@ -141,46 +141,24 @@ template<typename CharacterType> constexpr bool isTabOrSpace(CharacterType chara
 // Infra's "ASCII whitespace" <https://infra.spec.whatwg.org/#ascii-whitespace>
 template<typename CharacterType> constexpr bool isASCIIWhitespace(CharacterType character)
 {
-    // Histogram from Apple's page load test combined with some ad hoc browsing some other test suites.
-    //
-    //     82%: 216330 non-space characters, all > U+0020
-    //     11%:  30017 plain space characters, U+0020
-    //      5%:  12099 newline characters, U+000A
-    //      2%:   5346 tab characters, U+0009
-    //
-    // No other characters seen. No U+000C or U+000D, and no other control characters.
-    // Accordingly, we check for non-spaces first, then space, then newline, then tab, then the other characters.
-
-    return character <= ' ' && (character == ' ' || character == '\n' || character == '\t' || character == '\r' || character == '\f');
+    return character == ' ' || character == '\n' || character == '\t' || character == '\r' || character == '\f';
 }
 
-template<typename CharacterType> constexpr bool isJSONOrHTTPWhitespace(CharacterType character)
+template<typename CharacterType> constexpr bool isJSONOrHTTPOrXMLWhitespace(CharacterType character)
```

Comment:
Not a fan of this laundry-list function name. Some ideas: `isMinimalASCIIWhitespace`, `isNonFFASCIIWhitspace`, `isBaseASCIIWhitespace`. None of those are great but all seem better to me than the list of specifications.

Reply:
Thanks for the review! I tend to agree with your nit, but when I suggested a name along these lines (`isASCIIWhitespaceWithoutFormFeed()`) I got this feedback:

> It tells the reader what the function does mechanically, but not conceptually. How would one know if it’s used appropriately when looking at the code?

Reply:
I think that advice is misguided. Asking this header to tell the future about every specification that might want to use this does not seem right. If we really want do something like that, then we should just have separate inline functions with the name of each specification that all call the same underlying function and have a comment in each explaining why.

Reply:
Another idea is to rename as we import it into another namespace. This might work really well if it compiles because it might work better when you pass the function as a parameter to a template.
```
namespace JavaScriptCore { using isJSONSpace = isReducedASCIIWhitespace; }
```

Reply:
If that works, it might work within a single namespace too.